### DISCUSSION TITLE
Welcome to Navidrome Discussions!

### DISCUSSION THREAD

**Discussion by deluan** [navidrome/navidrome#2230] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by deluan** [comment]
Hi everyone,

I'm Deluan, the creator and main developer of Navidrome. I started Navidrome in 2016 as a personal project to learn the Go programming language, but it was in 2019 that I decide to make it a useful server for other music enthusiasts, incorporating a UI and making it compatible with as many Subsonic clients as possible.

Please don't hesitate to use this forum to ask any questions or share your thoughts on Navidrome.

**Reply by digua-cn** [reply]
Hello, I really like Navidrome and I want to make some simple modifications to generate a Docker image. How can I achieve this?

**Comment by praveensg8** [comment]
My navidrome library is not updating. its still spinning even after of  24hours. I have a around 14 000l inside my  external HDD which is  added as my music folder in  navidrome. It shows scann only 224 files.
Please advise how I cant updated my folder to navidrom quickly

**Comment by matakk** [comment]
I like this sample server ,but  the player does support speed 2 to play audio ? I do not play music ,just play some audio ,like book or talk .

**Reply by metalheim** [reply]
Navidromes own webui doesn't, but some 3rd party clients do (I know symfonium does, potentially others too)

**Comment by zahiraffan** [comment]
Hello! I appreciate this project! It's been fun setting up. I was wondering if there is an ETA on when we will get the next update (ie can we expect it before the year ends or will we need to wait until after the new year)?

**Comment by Samuelyilma** [comment]
I’m seriously impressed with Navidrome! I can’t believe I didn’t discover it sooner — it’s fast, beautiful, and works perfectly. I hope it continues evolving and could one day be a real alternative to Spotify.

One suggestion: for publicly hosted instances, it would be amazing if users could create their own accounts freely, making it more social and accessible.

Keep up the fantastic work!

**Comment by JadsonSiq98** [comment]
Eu sou Jadson Siqueira, professor de música e trombonista, movido pela convicção de que a educação musical transforma pessoas e comunidades. Atuo entre a sala de aula, a banda marcial e a criação de projetos culturais com visão estratégica, sempre buscando excelência, clareza e impacto real.

Tenho perfil prático e inovador, valorizo organização, estética profissional e soluções que funcionem no mundo real, seja numa partitura bem editada, num projeto bem estruturado ou numa apresentação que comunique propósito. Trago a tradição musical comigo, mas não fico preso a ela. Gosto de evoluir, otimizar e deixar tudo melhor do que encontrei.

Sou exigente porque acredito no potencial das pessoas e dos projetos que lidero. Para mim, resultado não é acaso. É trabalho bem pensado, consistência e visão de futuro.
